### PR TITLE
Parsing stdlib docs by AST parser

### PR DESCRIPTION
**Pull request by Overcastan**
Documentation for `stdlib` is now parsed by AST parser, and not by `stdlib_docs` parser

### CODE REVIEW

File: stdlib/stdlib_docs.rs [hunk 1]
```diff
@@ -1,125 +1,80 @@
...
+                        "| {} | {} |\n",
+                        func_full_name,
+                        proc.docs
+                            .clone()
+                            .unwrap()
+                            .replace('|', "\\|")
+                            .replace('\n', "<br /><br />")
+                    );
+                    f.write_all(func_output.as_bytes())
+                        .expect("unable to write func to writer");
```
Comment (bobbinth) [inline stdlib/stdlib_docs.rs:64]: We should probably put this logic into a helper function. This function could take a reference to a write and a `Procedure` struct and write the procedure's doc comments into the writer.

**Review by bobbinth — CHANGES_REQUESTED** [review]
Looks good! Thank you! I left a few small comments inline.

File: stdlib/stdlib_docs.rs [hunk 2]
```diff
@@ -1,182 +1,79 @@
...
-
-            // Render modules into markdown
-            f.write_all(module.comments.join("\n").as_bytes())
-                .expect("unable to write module comments");
             let header = format!(
                 "\n## {}\n| Procedure | Description |\n| ----------- | ------------- |\n",
                 ns
             );
 
             f.write_all(header.as_bytes())
```
[the remark this answers is not in the document]
Comment (Overcastan) [reply stdlib/stdlib_docs.rs:31]: Maybe move `write_proc_docs()` to the `MarkdownRenderer` too?

**Review by bobbinth — APPROVED** [review]
All looks good! Thank you!

**Pull request merged**